lib/scan_beacon/bluez_advertiser.rb

```diff
@@ -1,60 +1,42 @@
 module ScanBeacon
-  class BlueZAdvertiser
+  class BlueZAdvertiser < GenericIndividualAdvertiser
  
-    attr_accessor :beacon, :parser, :ad, :addr
+    attr_accessor :addr
 
     def initialize(opts = {})
       @device_id = opts[:device_id] || BlueZ.devices.map {|d| d[:device_id]}[0]
       raise "No available devices" if @device_id.nil?
       BlueZ.device_up @device_id
-      @addr = @initial_addr = BlueZ.devices.find {|d| d[:device_id] == @device_id}[:addr]
-      self.beacon = opts[:beacon]
-      self.parser = opts[:parser]
-      self.parser ||= BeaconParser.default_parsers.find {|parser| parser.beacon_type == beacon.beacon_types.first}
+      addr = @initial_addr = BlueZ.devices.find {|d| d[:device_id] == @device_id}[:addr]
```

Comment:
yeah.  it probably doesn't make a difference but I figured there is an accessor, and I wanted to be consistent with the way it was accessed in the start method